tests/monotouch-test/AVFoundation/AVAudioConverterPrimeInfoTest.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+#if XAMCORE_2_0
+using Foundation;
+using AVFoundation;
+#else
+using MonoTouch.AVFoundation;
+using MonoTouch.Foundation;
+#endif
+using NUnit.Framework;
+namespace MonoTouchFixtures.AVFoundation {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class AVAudioConverterPrimeInfoTest {
+	
+		[Test]
+		public void ConstructorTest ()
+		{
+			uint leading = 2;
+			uint trailing = 30;
+			
+			var info = new AVAudioConverterPrimeInfo (leading, trailing);
+			
+			Assert.AreEqual (leading, info.LeadingFrames);
```

Comment:
if there's more than one `Assert` in the test method **always** include a message, e.g. the property name `LeadingFrames`. That's useful when stack traces are not available or imprecise.